src/screens/ShoppingCart/index.tsx

```diff
@@ -0,0 +1,133 @@
+import React, {useEffect, useState, useCallback, useMemo} from 'react';
+import {StackNavigationProp} from '@react-navigation/stack';
+// import {RootStackParamList} from '../../types/StackNavigationType';
+import {CartType} from '@/types/OrderType';
+import {Alert, Text} from 'react-native';
+import PaymentSummary from '@/components/orderPage/PaymentSummary';
+import {getCartHistory} from '@/apis/Cart';
+import S from './ShoppingCartScreen.style';
+// type Props = {
+//   navigation: StackNavigationProp<RootStackParamList, 'Cart'>;
+// };
+import MarketInfo from '@/components/CartPage/MarketInfo';
+import {RootStackParamList} from '@/types/StackNavigationType';
+
+type Props = {
+  navigation: StackNavigationProp<RootStackParamList, 'Home'>;
+};
+
+// TODO: navigation: Cart 추가
+const ShoppingCartScreen = ({navigation}: Props) => {
+  // TODO: 결제 페이지로 이동 props 버튼
+  const [cartData, setCartData] = useState<CartType | null>(null);
+
+  const fetchDummyData = useCallback(async () => {
+    const res = await getCartHistory();
+    if (!res) {
+      Alert.alert('장바구니내역받아오기실패.');
+      return;
+    }
+    setCartData(res[0]);
+  }, []);
+
+  const {originalPrice, discountPrice} = useMemo(() => {
+    if (!cartData) {
+      return {originalPrice: 0, discountPrice: 0};
+    }
+
+    return cartData.products.reduce(
+      (acc, cur) => ({
+        originalPrice: acc.originalPrice + cur.originalPrice * cur.count,
+        discountPrice: acc.discountPrice + cur.discountPrice * cur.count,
+      }),
+      {originalPrice: 0, discountPrice: 0},
+    );
+  }, [cartData]);
+
+  const updateProductCount = (id: number, changeTerm: number) => {
+    if (!cartData) return;
+    setCartData(prevCartData => {
+      if (!prevCartData) return prevCartData;
+
+      const updatedProducts = prevCartData.products
+        .map(product =>
+          product.id === id
+            ? {...product, count: Math.max(0, product.count + changeTerm)}
+            : product,
+        )
+        .filter(product => product.count > 0);
+      console.log(cartData);
+      return {...prevCartData, products: updatedProducts};
+    });
+  };
+
+  const onPressStore = (marketId: number) => {
+    navigation.navigate('Detail', {
+      screen: 'Market',
+      params: {marketId},
+    });
+  };
+
+  useEffect(() => {
+    fetchDummyData();
+  }, [fetchDummyData]);
+
+  return (
+    <S.CartPage>
+      <S.ScrollView>
+        {cartData ? (
```

Comment:
```suggestion
        {!cartData && cartData.length !== 0 ? (
```

서버에서 받아온 데이터가 0일때도 장바구니가 비어있습니다 가 나와야 하니 조건추가가 필요할 것같아여